src/Command/ImportTerritorialCouncilFromCSVCommand.php

```diff
@@ -0,0 +1,124 @@
+    private $referentTagRepository;
+    /** @var SymfonyStyle */
+    private $io;
+
+    public function __construct(EntityManagerInterface $em, ReferentTagRepository $referentTagRepository)
+    {
+        $this->em = $em;
+        $this->referentTagRepository = $referentTagRepository;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->addArgument('file', InputArgument::REQUIRED, 'CSV file with territorial councils')
+        ;
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->io->text('Start importing territorial councils');
+
+        $territorialCouncils = $this->parseCSV($input->getArgument('file'));
+
+        $this->io->progressStart(\count($territorialCouncils));
+
+        $i = 0;
+        $batchSize = 10;
+        foreach ($territorialCouncils as $data) {
+            ++$i;
+            if (null == $data['name']) {
+                $this->io->warning('Territorial council without name cannot be added.');
+
+                continue;
+            }
+
+            /** @var TerritorialCouncil $territorialCouncil */
+            if ($territorialCouncil = $this->em->getRepository(TerritorialCouncil::class)->findOneBy(['name' => $data['name']])) {
+                $territorialCouncil->clearReferentTags();
+            } else {
+                $territorialCouncil = new TerritorialCouncil($data['name'], $data['codes']);
+            }
+
+            $tags = explode(',', $data['tag_code']);
+            foreach ($tags as $tag) {
+                $tag = trim($tag);
+                if (!($referentTag = $this->referentTagRepository->findOneByCode($tag))) {
+                    $this->io->warning(sprintf('ReferentTag with code (%s) has not been found.', $tag));
+                } else {
+                    $territorialCouncil->addReferentTag($referentTag);
+                }
+            }
+            $this->em->persist($territorialCouncil);
+
+            if (0 === ($i % $batchSize)) {
+                $this->io->progressAdvance($batchSize);
+                $this->em->flush();
+                $this->em->clear();
+            }
+        }
+
+        $this->io->progressFinish();
+        $this->em->flush();
+        $this->em->clear();
+
+        $this->io->writeln('');
+        $this->io->success('Done');
+    }
+
+    private function parseCSV(string $filename): array
+    {
+        $rows = [];
+        if (false === ($handle = @fopen($filename, 'r'))) {
+            throw new FileNotFoundException(sprintf('%s not found', $filename));
+        }
+
+        $firstline = true;
+
+        while (false !== ($data = fgetcsv($handle, 10000, ','))) {
+            if ($firstline) {
+                $firstline = false;
+
+                continue;
+            }
```

Comment:
If you want, to remove this if and the `$firstline` variable, you can call `fgetcsv` before the foreach, and do nothin with it. This will read the first line, and skip to the next one.